CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Mapbox welcomes participation and contributions from everyone.
 - Fixed the `NavigationView` issue where route arrows are being rendered above navigation puck after device rotation. [#6747](https://github.com/mapbox/mapbox-navigation-android/pull/6747)
 - Fixed an issue where a refresh of the routes or a change in alternatives that occurred while user was off-route would call `RerouteController#interrupt` and interrupt the potentially ongoing reroute process without recovery. [#6719](https://github.com/mapbox/mapbox-navigation-android/pull/6719)
 - Improved OpenLR matching rate. Updates take lowest FRC to next point into account during path generation. [#6750](https://github.com/mapbox/mapbox-navigation-android/pull/6750)
+- Changed billing behavior for EV routes: now new session will not be started in case an EV reroute or alternative route has different chargers. [#6703](https://github.com/mapbox/mapbox-navigation-android/pull/6703)
```

Comment:
Let's skip this entry. EV routes are still in preview and not supported as stable.

Reply:
Functional changes make sense but billing related ones are irrelevant at this point.